dataprofiler/profilers/json_encoder.py

```diff
@@ -24,9 +25,12 @@ def default(self, to_serialize):
             (
                 base_column_profilers.BaseColumnProfiler,
                 numerical_column_stats.NumericStatsMixin,
+                profiler_options.BaseOption,
             ),
         ):
             return {"class": type(to_serialize).__name__, "data": to_serialize.__dict__}
+        elif isinstance(to_serialize, set):
+            return list(to_serialize)
```

Comment:
was this necessary?

Reply:
nvm, i'm sad.

Reply:
need for `stop_words` in `TextProfilerOptions`

Reply:
We would need to convert back in that class if we don't.